### DISCUSSION TITLE
no #pragma handler registered for type MIME

### DISCUSSION THREAD

**Discussion by calonx** [WerWolv/ImHex#500] [Get Help] [answered]
Hello! Just discovered this tool today and am excited about the possibilities with the pattern language. However, I'm getting this error for the few default patterns I've tried to load. Am I missing something?

![Screenshot_2022-04-26_192733](https://user-images.githubusercontent.com/1431773/165414812-a0ed8d69-bff5-4534-90ce-2a3e9c6388cc.png)

In case it's not obvious from the screenshot, I'm running Windows (11, to be specific).

**Comment by WerWolv** [comment, marked as answer]
Hey, this was a bug, I just fixed it in 87e616ad230f16cb063fd35ba4fbe0db33530a14. Thanks for reporting

**Comment by calonx** [comment]
Thanks, @WerWolv!

Would you know whenabouts the bug was introduced? I'm not yet set up to build ImHex locally, so knowing that would allow me to grab an older release.

**Reply by WerWolv** [reply]
You probably rather want to grab the latest nightly release. They are linked in the readme
